client/platform/desktop/frontend/components/ImportDialog.vue

```diff
@@ -37,10 +46,15 @@ export default defineComponent({
 
     const sortedFpsOptions = computed(() => {
       const filteredOptions = FPSOptions
-        .filter((v) => v <= Math.round(argCopy.value.jsonMeta.originalFps));
-      filteredOptions.push(argCopy.value.jsonMeta.originalFps);
-      filteredOptions.sort((a, b) => a - b);
-      return uniq(filteredOptions);
+        .filter((v) => v.value <= Math.round(argCopy.value.jsonMeta.originalFps));
+      const videoFrameRateIndex = filteredOptions.findIndex((item) => item.value === -1);
+      if (videoFrameRateIndex !== -1) {
+        filteredOptions[videoFrameRateIndex] = {
+          text: `Video Frame Rate (${argCopy.value.jsonMeta.originalFps})`,
+          value: argCopy.value.jsonMeta.originalFps,
+        };
+      }
```

Comment:
These 7 lines could be replaced by:

```
filteredOptions.splice(-1, 0, { /* ... value ... */ })
```

This seems like a lot of logic just to change what you know is always the last element in the list.